addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -137,6 +137,48 @@ MockServer.include({
             grouped_activities: groupedActivities,
         };
     },
+    /**
+     * Simulate the 'channel_fold' route of of mail.channel
+     * In particular sends a notification on the bus
+     *
+     * @private
+     */
+     _mockChannelFold(args) {
+        const { state, uuid } = args.kwargs;
+        // FIXME: should use .env.messaging.currentPartner.id somehow
+        const partnerId = undefined;
```

Comment:
That's annoying... I think it's genuine `session.rpc()` first lines of code that extend user data in context, so that server is aware of partner id. This is not the case with mock server, and that's sucks...

Maybe we could add some user context in `web.test_env:session.rpc()` function before calling `providedRPC()`?

Anyway, all this is a problem of using a mock server that is too different from the real server, even in the params it gets. In mid/long-term, I would like that we use true server in our JS tests, so all of this is no longer an issue.

I would put brief mention that user context is not available in mock server. Also remove `FIXME`, because that's not a "fix me" in this LOC/function/mock server but it's a limitation of architecture in JS tests.